book/src/pil/expressions.md

```diff
@@ -201,5 +201,5 @@ let square_of = query |x| { let v = std::prover::eval(x); v * v };
 let new_bool = constr |x| { let x = new_wit(); x * (x - 1) = 0; x };
 // This is a pure function that only returns a constraint, but does not add it
 // to the global set of constraints.
-let constrain_to_bool: expr -> constr = |x| x * (x - 1) = 0;
+let constrain_to_bool: expr -> Constr = |x| x * (x - 1) = 0;
```

Comment:
I was momentarily confused by the name `constrain_to_bool`, as we have a very similarly named type `Constr` and this follows the naming for "conversion" functions.
Maybe this could be called something else? Maybe `ensure_bool`? Or even `bool_constraint`?